utils/fault.py

```diff
@@ -0,0 +1,25 @@
+import wpilib
+from enum import Enum
+
+
+class ErrorType(Enum):
```

Comment:
```suggestion
class Severity(Enum):
```

Pour être conséquent avec le nom de ton paramètre dans le constructeur de `Fault`.